Homework13/Tests/WebTableTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿using Homework13.Data.Constants;
+using Homework13.PageObjects;
+using NUnit.Framework;
+
+namespace Homework13.Tests
+{
+    public class WebTableTests : BaseTest
+    {
+        const string firstName = "Rita";
```

Comment:
Naming for consts in class context - starts with upper case letter.
But let's improve and please create a class to get random string of length you pass in parameter